src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -325,4 +326,37 @@ avoiding peak traffic in order to address both inconsistencies diagnostic and fi
 
 #### How to solve
 
-Under development: Task for solving mailbox inconsistencies
\ No newline at end of file
+Under development: Task for solving mailbox inconsistencies
+
+## Mail Queue
+
+An email queue is a mandatory component of SMTP servers. It is a system that creates an array of emails that are waiting to be processed for delivery. Email queuing is a form of Message Queuing – an asynchronous service-to-service communication. A message queue is meant to decouple a producing process from a consuming one. An email queue decouples the sender from the recipient. It allows them to communicate without being connected. As such, the queued emails wait for processing until the recipient is available to receive them. As James is an Email Server, it also supports mail queue as well.
+
+Sometimes you might need to check mail queue to make sure all emails are delivered properly. At first, you need to know why email queues get clogged. Here are the two core reasons for that:
+
+- Exceeded volume of emails
+- Spam-related issues
+
+### Browse Filtering
```

Comment:
This section should actually be named `Fine tune configuration for RabbitMQ mailQueue`, it should be in `james-project/src/site/xdoc/server/config-rabbitmq.xml` and we should simply link to it to keep this guide short.